exist-core/src/main/java/org/exist/xquery/functions/math/NoParamFunctions.java

```diff
@@ -65,15 +64,6 @@ public int getDependencies() {
          */
     public Sequence eval(Sequence[] args, Sequence contextSequence) throws XPathException {
 
-    	if (context.getProfiler().isEnabled()) {
```

Comment:
Did you mean to remove this?

Reply:
yes, as BasicFunction implements this functionality already.